server.py

```diff
@@ -57,48 +70,68 @@ def Ping(self):
+
     def Close(self):
+        '''Closes the server (which then closes the clients).'''
+
         self._server.close(self.gameid)
+
 class BoxesServer(PodSixNet.Server.Server):
  
     channelClass = ClientChannel
     def __init__(self, *args, **kwargs):
         PodSixNet.Server.Server.__init__(self, *args, **kwargs)
-        self.games = []
-        self.queue = None
-        self.currentIndex=0
+        self.games: list = []
+        self.current_game: Game = None
+        self.currentIndex: int = 0
 
     def Connected(self, channel, addr):
+        '''
+        Matches players with each other as they connect.
+        Creates Game objects as needed.
+        '''
+
         print('new connection:', channel)
-        if self.queue==None:
+
+        if self.current_game==None:
             self.currentIndex+=1
             channel.gameid=self.currentIndex
-            self.queue=Game(channel, self.currentIndex)
+            self.current_game=Game(channel, self.currentIndex)
         else:
             channel.gameid=self.currentIndex
-            self.queue.player1=channel
+            self.current_game.player1=channel
             print("Connecting to BOTH players")
-            self.queue.player0.Send({"action": "connectedPlayers","player":0, "gameid": self.queue.gameid})
-            self.queue.player1.Send({"action": "connectedPlayers","player":1, "gameid": self.queue.gameid})
-            self.games.append(self.queue)
-            self.queue=None
+            self.current_game.player0.Send({"action": "connectedPlayers","player":0, "gameid": self.current_game.gameid})
+            self.current_game.player1.Send({"action": "connectedPlayers","player":1, "gameid": self.current_game.gameid})
+            self.games.append(self.current_game)
+            self.current_game=None
 
         channel.count = 0
         channel.times = []
         self.start_time = 0
         # channel.Ping() #ping line
 
-    def readyStatus(self, data):
+    def readyStatus(self, data: dict):    # Redundant? Message commands already
+                                    # Message commands already exist in Game
+        '''
+        Passes on message to game instance.
+        '''
         game = [a for a in self.games if a.gameid==data["gameid"]]
         if len(game)==1:
             game[0].readyStatus(data)
 
-    def recieveCommand(self, data):
+    def recieveCommand(self, data: dict): # Redundant? Message commands already
+                                    # Message commands already exist in Game
+        '''
+        Passes on message to game instance.
+        '''
+        # why use set building notation?
         game = [a for a in self.games if a.gameid==data["gameid"]]
+
         if len(game)==1:
             game[0].recieveCommand(data)
 
-            print("Time taken to finish server operations", time() - self.start_time)
+            # print("Time taken to finish server operations", time() - self.start_time)
```

Comment:
Oops, did not mean to comment out. 
I think we should vote on using f-strings, since they format much more cleaner than C-style string formatting.

Reply:
Oh yeah f-strings look great, so it will just evaluate anything within the {}? Assuming that whatever gets evaluated returns a string or int?